content/projects/jam/index.adoc

```diff
@@ -10,49 +10,46 @@ links:
   gitter: 'jenkinsci/advocacy-and-outreach-sig'
 ---
 
-Jenkins Area Meetups (JAMs) are local meetups intended to bring Jenkins users
-and contributors together for socializing and learning.
-JAMs are organized by local Jenkins community members who have a passion for
-sharing new Jenkins concepts, patterns and tools. JAMs can be found
-link:https://www.meetup.com/pro/jenkins/[around the world], and if there isn't a
-JAM in your city, you could be the one to start it!
+There has been a few updates to the Jenkins Area Meetups. First, the Jenkins Area Meetup Pro account, which JAMs has been under, has been rebranded to reflect our new sponsor, the Continuous Delivery Foundation (CDF). We notified JAM organizers about this transition in early October 2019. 
 
-Jenkins Area Meetups are driven by local organizers but receive support from
-the Jenkins project via swag (stickers, etc), promotion, and help bootstrapping
-and operating the meetup group.
+JAM organizers are welcome to transition their meetup to CDF specific branding (if you would also like to update your meetup name we offer the following naming convention Project Name City Name Meetup or CI/CD City Name Meetup). By no means JAMs are under any obligation to update the naming convention or the branding. We are giving organizers the freedom to choose. For more info, please visit us at https://www.meetup.com/pro/cicd-cdf or drop us a line at https://www.meetup.com/pro/cicd-cdf.
+ 
+CI/CD Meetups (formerly JAMs) are local meetups intended to bring CI/CD users and contributors together for socializing and learning. CI/CD Meetups are organized by local community members who have a passion for sharing new CI/CD concepts, patterns and tools. We are also welcoming community members who are interested in other CI/CD projects such as Spinnaker, Tekton, and Jenkins X to join our network. CI/CD Meetups can be found link:https://www.meetup.com/pro/cicd-cdf[around the world], and if there isn’t a CI/CD Meetup in your city, you could be the one to start it! Apply link:https://docs.google.com/forms/d/e/1FAIpQLSeaoOkc-1FAcyYbD7TYOjJoiXbX7cQTArRACsps2g1VG7xgew/viewform[here]! 
```

Comment:
This is critical, because the Google form does not fully document the requirements to meetups

```suggestion
CI/CD Meetups (formerly JAMs) are local meetups intended to bring CI/CD users and contributors together for socializing and learning. CI/CD Meetups are organized by local community members who have a passion for sharing new CI/CD concepts, patterns and tools. We are also welcoming community members who are interested in other CI/CD projects such as Spinnaker, Tekton, and Jenkins X to join our network. CI/CD Meetups can be found link:https://www.meetup.com/pro/cicd-cdf[around the world], and if there isn’t a CI/CD Meetup in your city, you could be the one to start it! See the application guidelines link:https://github.com/cdfoundation/meetups#how-to-apply[here]! 
```